app/actors/SitemapGeneratingActor.scala

```diff
@@ -0,0 +1,21 @@
+package actors
+
+import akka.actor.Actor
+import models.documentation.{Sitemap, Documentation}
+
+object SitemapGeneratingActor {
+
+  case class GenerateSitemap(documentation: Documentation)
+
+}
+
+class SitemapGeneratingActor extends Actor {
+  import SitemapGeneratingActor._
+
+  def receive = {
+    case GenerateSitemap(documentation) =>
```

Comment:
The sitemap could be cached based on an xor of the hashes of the translations - if that changes, regenerate the sitemap, otherwise, do nothing.